Markup of signup pages

MasatakaOba opened the pull request:
## WHAT
Markup of the pages for signup

## WHY
To register the information of users By themselves

<img width="495" alt="image" src="https://user-images.githubusercontent.com/27143270/54837556-088d4b00-4d0a-11e9-8022-33077145d172.png">

![image](https://user-images.githubusercontent.com/27143270/54837939-e811c080-4d0a-11e9-8ff0-af6fa249d265.png)

<img width="565" alt="image" src="https://user-images.githubusercontent.com/27143270/54838041-142d4180-4d0b-11e9-80a0-737b9648c197.png">

<img width="578" alt="image" src="https://user-images.githubusercontent.com/27143270/54838076-23ac8a80-4d0b-11e9-96b5-8e5b5a6b1ca4.png">

<img width="569" alt="image" src="https://user-images.githubusercontent.com/27143270/54838102-2e671f80-4d0b-11e9-8109-5b3b1401e7d3.png">

<img width="561" alt="image" src="https://user-images.githubusercontent.com/27143270/54838131-3b840e80-4d0b-11e9-8604-74fcf56d0e65.png">

Commit:
Change erb to haml

Commit:
markup the pages of signup

1.Add some images for payment_registration.
2.Add scss files for decorating pages
3.Add haml files for signup (index, new, adress_registration, sms_conformation,payment_registration,completion)

Commit:
edit error message

change some messages and decoraton places as the same as original site

Commit:
Refactor code of scss and haml files

yuki-uchida commented on app/assets/stylesheets/modules/_signup.scss:
```diff
@@ -1,10 +1,99 @@
+  justify-content: center;
+}
+
+.color-red{
```
この辺り、BEMに沿ってクラス命名を行なってアンバサンドを活用した省略形で記述してください。以降も同様です

MasatakaOba replied:
クラス命名再実施し、アンパサンド活用し記述しました

yuki-uchida commented on app/views/devise/registrations/edit.html.haml:
```diff
@@ -0,0 +1,36 @@
+  = render "devise/shared/error_messages", resource: resource
+  .field
+    = f.label :email
+    %br/
```
brではなくmargin paddingでデザインを整えてください

MasatakaOba replied:
すみません、担当外の部分を誤ってhaml化、githubにpushしてしまいました。
担当の方が別なので確認後編集依頼します。

yuki-uchida replied:
了解しました

Commit:
Rewrite code with BEM notation and Use ampersand

Some code move to common scss file because those  code was used at other pages which another member make.

yuki-uchida commented on app/assets/stylesheets/modules/_common.scss:
```diff
@@ -0,0 +1,67 @@
+  outline: 0;
+  }
+}
+ .has-error-text{
```
この辺りも

MasatakaOba replied:
失礼しました。
signupに移し、アンパサンドを使い、表示される要素配下に来るように記載しました。

yuki-uchida reviewed (changes requested):
もう少しだけBEMに沿って書きましょう

Commit:
edit error message decoration code

yuki-uchida commented:
LGTMです。お疲れ様でした
